### ISSUE TITLE
[Feature Request] 24 hour time format

### ISSUE THREAD

**Issue by senpl** [nacasha/flutter-things-todo#4]
It could be optional or just change. Anyway users should be able to see their date in 24 h format.

**Comment by nacasha** [comment]
Yeah, it would be nice if users can set their own preferences, I will consider it in the next release
